app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -24,3 +24,34 @@ export const NOTES_TABLE_COLUMN_DATA = [
     width: "70%",
   },
 ];
+
+export const NOTES = [
+  {
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    execution_status: "Getting Started",
+    status: "Created",
+    created_at: "2 hours ago",
+  },
+  {
+    title: "How to claim the warranty?",
+    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    execution_status: "Getting Started",
+    status: "Drafted",
+    created_at: "2 hours ago",
```

Comment:
Why's this is snake_case and not camelCase?
https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react

Reply:
Fixed it.